.github/workflows/delete-unused-canary-packages.yml

```diff
@@ -0,0 +1,114 @@
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            global-phrases-block
+            ads-block
+            alert-bar-block
+            article-body-block
+            alert-bar-content-source-block
+            article-tag-block
+            author-bio-block
+            byline-block
+            card-list-block
+            date-block
+            default-output-block
+            double-chain-block
+            extra-large-manual-promo-block
+            extra-large-promo-block
+            footer-block
+            full-author-bio-block
+            gallery-block
+            header-block
+            header-nav-chain-block
+            headline-block
+      - name: Delete many package oldest canaries pt 2
+        uses: JackHowa/delete-github-package-versions@v0.4.9
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            htmlbox-block
+            large-manual-promo-block
+            large-promo-block
+            lead-art-block
+            links-bar-block
+            masthead-block
+            medium-manual-promo-block
+            medium-promo-block
+            numbered-list-block
+            overline-block
+            quad-chain-block
+            related-content-content-source-block
+            results-list-block
+            right-rail-block
+            right-rail-advanced-block
+            search-results-list-block
+            section-title-block
+            share-bar-block
+            shared-styles
+            simple-list-block
+      - name: Delete many package oldest canaries pt 3
+        uses: JackHowa/delete-github-package-versions@v0.4.9
+        with:
+          version-pattern: "canary"
+          keep: 5
+          dry-run: false
+          version-query-order: "first"
+          # 20 packages max
+          names: |
+            single-chain-block
+            small-manual-promo-block
+            small-promo-block
+            subheadline-block
+            tag-title-block
+            text-output-block
+            textfile-block
+            top-table-list-block
+            triple-chain-block
+            video-player-block
+            video-promo-block
+            content-api-source-block
+            collections-content-source-block
+            author-content-source-block
+            search-content-source-block
+            site-hierarchy-content-block
+            story-feed-author-content-source-block
+            story-feed-query-content-source-block
+            story-feed-sections-content-source-block
+            story-feed-tag-content-source-block
+      - name: Delete many package oldest canaries pt 4
+        uses: JackHowa/delete-github-package-versions@v0.4.9
```

Comment:
Not that I don't want to keep you forever, but does it make sense to have this moved into WPMedia instead of hosting on your own github?

Reply:
hahah ya can do 😿 . I have a pr open with the maintainer so fingers-crossed they can merge my fork in. 

will update

Reply:
![never leave](https://user-images.githubusercontent.com/13948125/118043488-6c888880-b343-11eb-8993-1091a68372cc.gif)

Reply:
hahahah. awaiting approval from it-help-desk for the github marketplace

Reply:
updated 😿